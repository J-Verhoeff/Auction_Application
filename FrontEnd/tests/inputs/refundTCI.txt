login
admin
refund
buyer
refund
user
seller
refund
user
admin
-1
refund
user
admin
101
refund
user
admin
10
logout
exit