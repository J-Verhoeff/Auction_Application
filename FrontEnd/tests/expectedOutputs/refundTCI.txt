
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Buyer's username: Buyer not a user!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Buyer's username: Seller's username: Seller not a user!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Buyer's username: Seller's username: Amount of credit: Amount of credit must be between 0 and 999,999!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Buyer's username: Seller's username: Amount of credit: Insufficient amount of credit in Seller's account!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Buyer's username: Seller's username: Amount of credit: Successfully refunded buyer!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: Goodbye!
